accounts/forms.py

```diff
@@ -327,3 +319,18 @@ def __init__(self, *args, **kwargs):
                 }
         super(DeleteUserForm, self).__init__(*args, **kwargs)
 
+
+class EmailSettingsForm(forms.Form):
+    enabled_stream_emails = forms.BooleanField(
```

Comment:
I guess this is related to the main comment. Stream email preferences should be merged with these, agree (so stream emails should become a new email type).